chia/rpc/wallet_rpc_api.py

```diff
@@ -810,6 +811,28 @@ async def delete_unconfirmed_transactions(self, request):
                 await self.service.wallet_state_manager.tx_store.rebuild_tx_cache()
                 return {}
 
+    async def select_coins(self, request) -> Dict[str, List[Dict]]:
+        assert self.service.wallet_state_manager is not None
+
+        if await self.service.wallet_state_manager.synced() is False:
+            raise ValueError("Wallet needs to be fully synced before selecting coins")
+
+        amount = request["amount"]
+        if not isinstance(amount, uint64):
+            amount = uint64(amount)
+
+        wallet_id = request["wallet_id"]
+        if not isinstance(wallet_id, uint32):
+            wallet_id = uint32(wallet_id)
```

Comment:
You can just do `wallet_id = uint32(request["wallet_id"])` here right?

Same with the amount above.